engine/runtime/src/main/java/org/enso/interpreter/node/expression/builtin/io/PrintlnNode.java

```diff
@@ -91,17 +93,20 @@ private void print(PrintStream out, Object str) {
     out.println(str);
   }
 
+  @NeverDefault
   UnresolvedSymbol buildSymbol() {
     return UnresolvedSymbol.build("to_text", EnsoContext.get(this).getBuiltins().getScope());
   }
 
+  @NeverDefault
   InvokeCallableNode buildInvokeCallableNode() {
     return InvokeCallableNode.build(
         new CallArgumentInfo[] {new CallArgumentInfo()},
         InvokeCallableNode.DefaultsExecutionMode.EXECUTE,
         InvokeCallableNode.ArgumentsExecutionMode.PRE_EXECUTED);
   }
 
+  @NeverDefault
```

Comment:
Another [interesting @NeverDefault annotation](https://www.graalvm.org/truffle/javadoc/com/oracle/truffle/api/dsl/NeverDefault.html).


I am not sure why it is needed for a method that constructs a `Node` when constructing a `Node` is forbidden on fast path?

Reply:
`PrintlnNodeGen.create()` is `@NeverDefault`, but it seems that Truffle cannot determine that at this point, so one has to annotate even this `build()` method. We might avoid these complications by following Truffle's naming conventions for `create` methods.